src/components/atoms/ImageInput/ImageInput.scss

```diff
@@ -83,3 +83,8 @@ $upload-button-height: 150px;
     }
   }
 }
+
+.SpaceEditForm .AdminSection__label .ImageInput__container.mod--hidden {
```

Comment:
needed to overwrite basic height/padding of `container` component if logo/bannerImage is present